### PR TITLE
(feat) O3-3132: Ability to import JSON schemas into the form builder

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary
- This PR adds the ability to import a finished form into o3 instead of copying and pasting.

## Screenshots

https://github.com/openmrs/openmrs-esm-form-builder/assets/30952856/43e673cd-da6f-456e-8804-336a914f8775

## Related Issue
[Ticket](https://openmrs.atlassian.net/browse/O3-3132)

### CODE REVIEW

**Comment by pirupius** [comment]
@hadijahkyampeire was there a ticket for this? am curious if there was some file validation needed or it was only meant to copy it's contents and everything else else handled on the form builder level?

**Comment by hadijahkyampeire** [comment]
@pirupius there is a ticket but I am the one who created it, this is something we just talked about with @denniskigen at a higher level, so it is open to requirement changes.

**Review by denniskigen — COMMENTED** [review]
Mostly LGTM, @hadijahkyampeire, save for a few suggested changes. Thanks!

File: src/components/form-editor/form-editor.component.tsx [hunk 1]
```diff
@@ -265,8 +287,23 @@ const FormEditor: React.FC = () => {
...
-              <div>
+              <div className={styles.topBtns}>
+                {!schema ? (
+                  <FileUploader
+                    onChange={handleSchemaImport}
+                    labelTitle=""
+                    labelDescription=""
+                    buttonLabel={t('importSchema', 'Import schema')}
+                    buttonKind="ghost"
+                    size="md"
```
[the remark this answers is not in the document]
Comment (denniskigen) [reply src/components/form-editor/form-editor.component.tsx:298]: So the button is the same size as the others.

**Comment by hadijahkyampeire** [comment]
@denniskigen I have addressed the issues.

**Comment by denniskigen** [comment]
Thanks, @hadijahkyampeire!

**Review by denniskigen — APPROVED** [review]

**Pull request merged**